### PR TITLE
Update cluster-administration-overview.md

### PR DESCRIPTION
**Pull request by ASaiun**
remove the sentence "If you are using a guide involving Salt, see [Configuring Kubernetes with Salt](/docs/admin/salt/)."
 because the website can not be find ,it will return 404

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.12 Features: set Milestone to 1.12 and Base Branch to release-1.12
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> Please delete this note before submitting the pull request.

### CODE REVIEW

**Comment by mistyhacks** [comment]
Deploy preview for *kubernetes-io-master-staging* ready!

Built with commit 1f3d7a2fd76dc8c75aa6002ebc081f71af1f31fc

https://deploy-preview-9449--kubernetes-io-master-staging.netlify.com

File: content/en/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 1]
```diff
@@ -32,8 +32,6 @@ Before choosing a guide, here are some considerations:
 
 Note: Not all distros are actively maintained. Choose distros which have been tested with a recent version of Kubernetes.
 
-If you are using a guide involving Salt, see [Configuring Kubernetes with Salt](/docs/admin/salt/).
```
Comment (tengqm) [inline content/en/docs/concepts/cluster-administration/cluster-administration-overview.md]: The link can be replaced with /docs/setup/salt

Comment (ASaiun) [reply content/en/docs/concepts/cluster-administration/cluster-administration-overview.md]: ok

**Comment by tengqm** [comment]
@ASaiun Please sign the CLA in order to merge this.

**Pull request merged**